docs/build-build-with-polkadot.md

```diff
@@ -289,7 +288,7 @@ parachains.
 ink! has laid much of the groundwork for a new smart contract stack that is based on a Wasm virtual
 machine and compatible with Substrate chains.
 
-### It's still early
+### Smart Contract Parachains
 
 It’s still very early for smart contracts on Polkadot and the development is only now stabilizing.
 We are actively producing content to help developers get up to speed and will maintain the wiki with
```

Comment:
Separating the list into sections may be better suited.

Something like: 
`Parity Tech`
- ink!
- Substrate contracts pallet

and then a `Parachains` section
- Edgeware
- Moonbeam
- Acala
- Phala